packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -184,7 +184,16 @@ export async function copyProjectTemplateAndReplace(
     'WinUI3Version',
     winui3PropsPath,
   );
-
+  const winui2xPropsPath = require.resolve(
+    'react-native-windows/PropertySheets/WinUI.props',
+    {paths: [process.cwd()]},
+  );
+  const winui2xProps = readProjectFile(winui2xPropsPath);
```

Comment:
These are the same as above, you can remove these and just use the variables `winui3PropsPath` and `winui3Props` (can rename those if you feel like it)